### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by WindowsNinetyTen** [FIX94/Nintendont#1124]
* GamePad rumble
* Microphone emulation that can use a USB microphone, a Wii microphone, a Wii U microphone, the Wii U GamePad microphone and so forth
* GameCube-GBA Cable support for Wii U
* Stable 1019-block memory card emulation

**Comment by carnage702** [comment]
your free to make a pull request to add those features, but no one is working on nintendont to do that anymore.

Also gamecube gba cable support for wiiu is impossible, all gc controller adapters that exist dont capture the signal from gba cable adapter, all they do is turn gc controller signal into hid code to get gbalink cable code you will need the adapters to allow raw data transfer both ways which none of them do not even official or fake ones.

**Comment by WindowsNinetyTen** [comment]
Okay. I suppose that I shall have to try to add them myself.

Also, can Game Boy Player be emulated as well?

**Comment by carnage702** [comment]
> Okay. I suppose that I shall have to try to add them myself.
> 
> Also, can Game Boy Player be emulated as well? I know that a physical player cannot be attached but what about emulating the hardware?

why would you emulate gameboy games on nintendont... there are countless gbc/gba emulators for wii/wiiu they all have preety much perfect compatibility and features, what would you gain trying to cram an emulator inside nintendont? lol

**Comment by WindowsNinetyTen** [comment]
Right. After all, there is also the Wii U Virtual Console, so it's no big deal.

**Comment by WindowsNinetyTen** [comment]
How about an additional feature: Enable GameCube disc playback on Wii U via minidisc adaptor. It would work like this:

1. GameCube retail disc is put in a minidisc adaptor that would make it be the size of a full disc
2. Disc is inserted into console
3. Nintendont helps system to detect GameCube disc and the loader would show up on the disc channel

This can work for burned games also.

**Comment by WindowsNinetyTen** [comment]
I understand, I meant to suggest adding a GameCube disc decoder for Wii U by allowing this app to interact with the disc drive and unlock it, and have it read the discs with one of those adaptors that make it be the size of full discs.

**Comment by WindowsNinetyTen** [comment]
But wouldn't it be possible with a firmware mod for the drive?

**Comment by carnage702** [comment]
> But wouldn't it be possible with a firmware mod for the drive, which this app could be made to provide?

no you cant mod the drives at all ... like i said people cant even make the wiiu read regular dvd movies or blurays... let alone read that...

**Comment by WindowsNinetyTen** [comment]
Be that as it may, can the disc drive light at least be made to light up when running Nintendont?
